src/main.cpp

```diff
@@ -679,9 +679,6 @@ bool IsStandardTx(const CTransaction& tx, string& reason)
         else if ((whichType == TX_MULTISIG) && (!fIsBareMultisigStd)) {
             reason = "bare-multisig";
             return false;
-        } else if (txout.IsDust(::minRelayTxFee)) {
```

Comment:
By removing this check, this would allow multiple txouts with 0 balance.  Previously, the only time this was allowed was with a single OP_RETURN txout.